.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,77 @@
+name: Ci suite
+on: [push, pull_request]
+jobs:
+    symfony:
+        name: PHPUnit tests (PHP ${{ matrix.php-versions }})
+        runs-on: ubuntu-latest
+
+        # Docs: https://docs.github.com/en/actions/using-containerized-services
+        services:
+            postgres:
```

Comment:
Why is this needed for unit tests? Did you mean to name the file phpunit-tests?

Reply:
The PHPUnit tests also run functional tests, so it's not just "unit-tests", I will rename the file and job